codegen-renderers/src/test/resources/typescript/tsconfig.json

```diff
@@ -2,14 +2,15 @@
   "compilerOptions": {
     "outDir": "dist",
     "rootDir": "src",
-    "target": "es2018",
+    "target": "es2019",
     "types": ["node"],
     "module": "commonjs",
     "moduleResolution": "node",
-    "sourceMap": true,
+    "sourceMap": false,
```

Comment:
Was is intentional that you disabled source maps for this test project?

Reply:
actually for the moment we try to use babel for transpilation, and tsc for type checking, so we try to rely more on babel for emiting artifacts.
nonetheless we didn't succeed yet and maybe we ll have to re-enable it later